Backpack-SwiftUI/Calendar/Classes/CalendarMonthHeader.swift

```diff
@@ -0,0 +1,88 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+/// CalendarMonthHeader is a view that displays the month name and an optional accessory action.
+/// Also exposes a binding to the currently shown month.
+struct CalendarMonthHeader: View {
```

Comment:
Should this have a snapshot test as well, it looks like it contains some layout logic for different states

Reply:
It's also covered by the other snapshots! I dont think it needs its own!